src/ott/geometry/geometry.py

```diff
@@ -129,30 +129,58 @@ def mean_cost_matrix(self) -> float:
     tmp = self._masked_geom().apply_cost(self._n_normed_ones).squeeze()
     return jnp.sum(tmp * self._m_normed_ones)
 
+  @property
+  def std_cost_matrix(self) -> float:
+    r"""Standard deviation of all values stored in :attr:`cost_matrix`.
+
+    Uses the :meth:`~ott.geometry.Geometry.apply_square_cost` to remain
+    applicable to low-rank matrices, through the formula:
+
+    .. math::
+        \sigma^2=\frac{1}{nm}\left(\sum_{ij} C_{ij}^2 -
+        (\sum_{ij}C_ij)^2\right).
+
+    to output :math:`\sigma`.
+    """
+    tmp = self._masked_geom().apply_square_cost(self._n_normed_ones).squeeze()
+    return jnp.sqrt(
+        jax.nn.
```

Comment:
Can you please split this in a nicer way? The formatting is a bit off.

Reply:
yes, this was yapf. will correct